host/3.0/buster/amd64/java/java11/java11-composite.Dockerfile

```diff
@@ -0,0 +1,33 @@
+ARG JAVA_VERSION=11u7
+
+FROM mcr.microsoft.com/java/jre-headless:${JAVA_VERSION}-zulu-debian10-with-tools as jre
+FROM mcr.microsoft.com/dotnet/core/runtime-deps:3.1
+ARG HOST_VERSION
+
+EXPOSE 2222 80
+
+ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
+    HOME=/home \
+    FUNCTIONS_WORKER_RUNTIME=java \
+    DOTNET_USE_POLLING_FILE_WATCHER=true \
+    HOST_VERSION=${HOST_VERSION}
+
+COPY --from=runtime-image [ "/azure-functions-host", "/azure-functions-host" ]
+COPY --from=runtime-image [ "/workers/java", "/azure-functions-host/workers/java" ]
+COPY --from=jre [ "/usr/lib/jvm/zre-hl-tools-11-azure-amd64", "/usr/lib/jvm/zre-11-azure-amd64" ]
+
+## Using this method with ../ requires the docker build be run from the /out/ folder.
+## Alternative solution would be to copy the main files for sshd and start.sh to each lang folder
+COPY ../shared/sshd_config /etc/ssh/
+COPY ../shared/start.sh /azure-functions-host/
```

Comment:
`..` doesn't really work in `Dockerfile` as you mentioned in the comment the docker build command needs to run from the `./out` dir. `../shared` will resolve to `/shared` in the current directory

Reply:
Agreed.  Was just trying to think of a way to fix this.  I see two solutions. 
1: Move all output dockerfiles to the same base /generated/ dir
         Would allow us to just use the two config files in all the dirs instead of copying over lots of them
2: Copy configs to the language folders from a main copy in the /base/ folder
        Copying over this file into all the language folders would also simplify this problem. But, we would create lots of identical files.  Copying them all from the main copy in the /base/ folder would allow us to change all the configs at once similar to the base image.

Reply:
Chose to copy all configs over to the output folder during ./generate-composite.sh. Will push latest updates with these changes.